modules/microshift-creating-backups.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-creating-backups_{context}"]
+= Creating backups using the auto-recovery feature
+
+You can use the following procedure to create backups.
+
+[NOTE]
+====
+Creating backups require stopping {microshift-short}, you must determine the best time to stop {microshift-short}.
```

Comment:
Grammar - I'd add something here to join these two independent clauses:
```suggestion
Creating backups require stopping {microshift-short}, so you must determine the best time to stop {microshift-short}.
```

Reply:
done